test-requirements.txt

```diff
@@ -0,0 +1,2 @@
+odoo-addon-mrp_flattened_bom_xlsx@git+https://github.com/OCA/manufacture-reporting.git@refs/pull/112/head#subdirectory=setup/mrp_flattened_bom_xlsx
+odoo-addon-report_xlsx@git+https://github.com/OCA/reporting-engine.git@16.0#subdirectory=setup/report_xlsx
```

Comment:
Can you remove the test-requirements? dependencies are merged.

Reply:
Done